### PR TITLE
Rename KILT Protocol to "BOTLabs"

### PR DESCRIPTION
**Pull request by weichweich**
* The full node is provided by "BOTLabs GmbH" and not "KILT Protocol".
* Remove the WILT network. This will most likely never be used again.

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts [hunk 1]
```diff
@@ -383,7 +383,7 @@ export const prodParasPolkadot: Omit<EndpointOption, 'teleport'>[] = [
     paraId: 2086,
     providers: {
       Dwellir: 'wss://kilt-rpc.dwellir.com',
-      'KILT Protocol': 'wss://spiritnet.kilt.io/',
+      BOTLabs: 'wss://spiritnet.kilt.io/',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayPolkadot.ts:386]: As per the CI failure, we should just move this entry up to be alphabetical.

File: packages/apps-config/src/endpoints/testingRelayWestend.ts [hunk 2]
```diff
@@ -107,19 +107,6 @@ export const testParasWestend: Omit<EndpointOption, 'teleport'>[] = [
       color: '#03f3f3',
       logo: nodesKhalaSVG
     }
-  },
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testingRelayWestend.ts:110]: Just needs a removal of the actual (now unused) png file as well in the import on line 8.

**Review by jacogr — CHANGES_REQUESTED** [review]
Some small changes due to CI failures.

File: packages/apps-config/src/endpoints/productionRelayPolkadot.ts (hunk 1 above)
Comment (weichweich) [reply packages/apps-config/src/endpoints/productionRelayPolkadot.ts:385]: Thanks for the quick review and sorry for the sloppy mistakes. Should be fixed: [32356ea](https://github.com/polkadot-js/apps/pull/10032/commits/32356ea1b9ddbf0c58eb34b67f0ef0f79035b859)

File: packages/apps-config/src/endpoints/testingRelayWestend.ts (hunk 2 above)
Comment (weichweich) [reply packages/apps-config/src/endpoints/testingRelayWestend.ts:110]: Fixed in [39d90b5](https://github.com/polkadot-js/apps/pull/10032/commits/39d90b5a78fefcc69dddac3be163ec0aea2ea66e)

**Review by jacogr — APPROVED** [review]

**Pull request merged**